From afe1972b69c6a66a400cf54560e94ac4ad8823fa Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 12 Jan 2017 11:41:45 -0800
Subject: [PATCH] joyent/node-triton#163 "skipAffinityTests" option for
 test/config.json is not documented in test/config.json.sample Reviewed by:
 Julien Gilli <julien.gilli@joyent.com> Approved by: Julien Gilli
 <julien.gilli@joyent.com>

---
 test/config.json.sample | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/test/config.json.sample b/test/config.json.sample
index 957fa18..44e2e65 100644
--- a/test/config.json.sample
+++ b/test/config.json.sample
@@ -9,12 +9,18 @@
         "account": "joe.blow",
         "keyId": "de:e7:73:32:b0:ab:31:cd:72:ef:9f:62:ca:58:a2:ec",
         "insecure": false
-    }
+    },
 
     // Optional. Set this to allow the parts of the test suite that create,
     // update and destroy resources: instances, images, networks, etc.
     // This is essentially the "safe" guard.
-    "allowWriteActions": true
+    "allowWriteActions": true,
+
+    // Optional. Set to 'true' to skip affinity tests. Affinity tests assume
+    // the datacenter has multiple servers to which provisions can go. So if
+    // you are testing against COAL, for example, you'll need to set this
+    // to true.
+    "skipAffinityTests": false,
 
     // The params used for test provisions. By default the tests use:
     // the smallest RAM package, the latest base* image.
-- 
2.21.0

